plugins/gimp/file-jxl-save.h

```diff
@@ -6,15 +6,61 @@
 #ifndef PLUGINS_GIMP_FILE_JXL_SAVE_H_
 #define PLUGINS_GIMP_FILE_JXL_SAVE_H_
 
-#include <libgimp/gimp.h>
-
-#include "lib/jxl/base/status.h"
+#include "jxl/encode.h"
+#include "jxl/encode_cxx.h"
+#include "plugins/gimp/common.h"
 
 namespace jxl {
 
-Status SaveJpegXlImage(gint32 image_id, gint32 drawable_id,
-                       gint32 orig_image_id, const gchar* filename);
+bool save_dialog();
```

Comment:
Given that this and `JpegXlSaveOpts` don’t seem to be used outside of the `.cc` file, it would perhaps be preferable not to declare them in the header file, and to put them in an anonymous namespace to give them internal linkage. (The local naming convention would also be to name the function `SaveDialog`.)

Reply:
What about the naming of the other functions, like the gnome callback functions?  Would those become GuiOnChangeSuchAndSuch()?